molecule/yaml_config_validation/templates/wfly.yaml.config.j2

```diff
@@ -0,0 +1,15 @@
+# {{ ansible_managed }}
```

Comment:
There is this template already provided by the systemd role, why do you need to add it in the test?

Reply:
@rpelisse deleted this service file too, as in `molecule/yaml_config_validation/converge.yml` we are using [wildfly_systemd role](https://github.com/ansible-middleware/wildfly/pull/87/files#diff-3c6b85262ba2bd9cc71e0b570899586131639eabbe87a99c4ad66992007129d0R14-R18)